js/src/dag.js

```diff
@@ -312,7 +337,12 @@ module.exports = (common) => {
       })
     })
 
-    describe('.tree', () => {
+    describe.skip('.tree', function () {
```

Comment:
Why skip for .all?

Reply:
Sorry again :-/ Something is wrong with this PR. I ran it with both js-ipfs and js-ipfs-api. But after this change I will run it again.